classes/form/selectidp_buttons.php

```diff
@@ -54,6 +54,8 @@ public function definition() {
         $mform->addElement('hidden', 'wants', $wants);
         $mform->addElement('checkbox', 'rememberidp' , '', get_string('rememberidp', 'auth_saml2'));
 
+        $mform->setType('wants', PARAM_RAW);
```

Comment:
Can you please also move this up and fix the whitespace so that each form element is grouped in the code

Reply:
Please move this up to where it should be

Reply:
Have added this now in its own commit